### PR TITLE
test: testing import and export records

### PR DESCRIPTION
**Pull request by jbenito3**
* Adds a test to load all demo records and then export them to the
  available formats. (related to invenio-records/#169)

### CODE REVIEW

File: tests/unit/test_invenio_app_ils.py [hunk 1]
```diff
@@ -31,3 +31,40 @@ def test_version():
...
+                  for x
+                  in PersistentIdentifier.query.filter(
+                    PersistentIdentifier.pid_type == 'recid'
+                  ).all()
+                  ]
+        with app.test_client() as client:
+            for recid in recids:
+                for fmt in export_formats:
+                    res = client.get(
+                        "/record/{0}/export/{1}".format(recid, 'marcxml')
```
Comment (ntarocco) [inline tests/unit/test_invenio_app_ils.py]: I think that you forgot the param here:
`"/record/{0}/export/{1}".format(recid, 'marcxml')`
should be
`"/record/{0}/export/{1}".format(recid, fmt)`
?

**Review by ntarocco — CHANGES_REQUESTED** [review]

File: tests/unit/test_invenio_app_ils.py (hunk 1 above)
Comment (jbenito3) [reply tests/unit/test_invenio_app_ils.py]: damned! well seen

Comment (jbenito3) [reply tests/unit/test_invenio_app_ils.py]: fixed! thanks :)

**Review by ntarocco — APPROVED** [review]

**Comment by jbenito3** [comment]
Test won't pass until invenio-records is released (due to inveniosoftware/invenio-records#180).

**Review by drjova — APPROVED** [review]

**Pull request merged**